bindgen/codegen/mod.rs

```diff
@@ -4487,21 +4504,71 @@ pub(crate) fn codegen(
             result.push(dynamic_items_tokens);
         }
 
+        if !result.items_to_serialize.is_empty() {
+            match utils::serialize_items(&result, &context) {
+                Ok(()) => (),
+                Err(err) => warn!("Could not serialize C items: {}", err),
```

Comment:
I think this should propagate up -- the build silently succeeding when stuff isn't generated is very confusing.